MatrixSDK/MXRestClient.m

```diff
@@ -5766,6 +5767,14 @@ - (MXHTTPOperation*)relationsForEvent:(NSString*)eventId
     {
         parameters[@"from"] = from;
     }
+    if (direction == MXTimelineDirectionForwards)
+    {
+        parameters[@"dir"] = @"f";
+    }
+    else
+    {
+        parameters[@"dir"] = @"b";
```

Comment:
These should be moved to the top of the file as constants.

Reply:
Makes sense.